contrib/go/3rdparty/go/github.com/golang/protobuf/proto/BUILD

```diff
@@ -0,0 +1,5 @@
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+go_remote_library(rev='v1.1.0')
```

Comment:
Note - this library will need to be published to https://github.com/pantsbuild/binaries

Reply:
I'm probably ok with hosting this on pantsbuild/binaries, but are there other options here? A consumer of this library will have a {{go}} environment bootstrapped by pants, so it seems like `go get` into a directory under `~/.cache/pants` (ie get_options().pants_bootstrapdir) would work ok too.

Reply:
Now that we switch to a `Subsystem`, I'll close the PR against `binaries` as that is no longer needed.